lib/scope.js

```diff
@@ -765,20 +754,20 @@ module.exports = {
   },  // Ends scope.getMatchingRows()
 
 
-  setVariable: async function ( scope, { field, var_data, from_table }) {
+  setVariable: async function ( scope, { field, var_data }) {
     /* Returns true if the field was set.
     *
     * TODO: Will the selector ever change the second time around?
     *   If so, we need to store the handles before this.
     * 
     * @returns bool
    */
-    let was_set = false;
+    let row_used = null;
```

Comment:
Hmm... I'm not sure why these changes are showing up in this diff. I thought these were from another MR that's already merged.

Reply:
That's disconcerting. Not sure how to track that down...